File: app/src/main/java/social/entourage/android/api/model/map/LastMessageAuthor.java
package social.entourage.android.api.model.map;

import com.google.gson.annotations.SerializedName;

import java.io.Serializable;

/**
 * Created by mihaiionescu on 13/10/16.
 */

public class LastMessageAuthor implements Serializable{

    private static final long serialVersionUID = 8632217230061284659L;
    
    @SerializedName("first_name")
    private String firstName;

    @SerializedName("last_name")
    private String lastName;

    public String getFirstName() {
        return firstName;
    }

    public void setFirstName(final String firstName) {
        this.firstName = firstName;
    }

    public String getLastName() {
        return lastName;
    }

    public void setLastName(final String lastName) {
        this.lastName = lastName;
    }

}

File: app/src/main/java/social/entourage/android/api/model/LastMessageAuthor.kt
package social.entourage.android.api.model

import com.google.gson.annotations.SerializedName
import java.io.Serializable

/**
 * Created by mihaiionescu on 13/10/16.
 */
class LastMessageAuthor : Serializable {
    @SerializedName("first_name")
    var firstName: String? = null

    @SerializedName("last_name")
    var lastName: String? = null

    @SerializedName("display_name")
    var displayName: String? = null

    @SerializedName("id")
    var authorId: Int = 0

    companion object {
        private const val serialVersionUID = 8632217120061284659L
    }
}